tests/fixtures/gregor/README.md

```diff
@@ -8,9 +8,16 @@ To run integrations tests in `tests/integration/gregor`:
 ```bash
 export GCS_OAUTH_TOKEN=$(gcloud auth application-default print-access-token)
 ```
-2. **Locate phenotype table**: Download a GREGoR formatted phenotype table as tsv from AnVIL platform, then either
-   1. Copy that file to `tests/fixtures/gregor` as  `phenotypes.tsv` or `phenotypes.csv` or...
-   2. `export PHENOTYPE_TABLE=<ABSOLUTE_PATH_TO_PHENO_TABLE>`
+2. **Locate phenotype table**: There are two ways to do this depending on your testing environment
+   1. If working in a Terra workspace,
+      1. Import the phenotype table from the [GREGoR U08 Workspace](https://app.terra.bio/#workspaces/gregor-dcc/GREGOR_COMBINED_CONSORTIUM_U08/data) in your workspace.
+      2. Then, the tests will automatically look for the table titled "phenotype" and load it.
+   2. If not in Terra,
+      1. Download the phenotype table from the [GREGoR U08 Workspace](https://app.terra.bio/#workspaces/gregor-dcc/GREGOR_COMBINED_CONSORTIUM_U08/data) as a tsv from AnVIL platform
+      2. Ensure the file name is `phenotype.tsv` or `phenotype.csv`
```

Comment:
should be plural ("phenotypes"). Also if I'm looking at the fixture code correctly, it looks like it will only check for a TSV, not a CSV

Reply:
updated to load in "phenotype.tsv" singular so that user doesn't have to change it